### PR TITLE
Fix for REGISTRY-3324

### PR DESCRIPTION
**Pull request by Malintha**

### CODE REVIEW

File: modules/es-extensions/publisher/extensions/app/greg-diff/themes/default/js/diff.js [hunk 1]
```diff
@@ -51,12 +51,12 @@ $('.container-fluid').resize(function(){
  */
 function setViewPanelsHeight(){
     if($(window).width() > 768){
-        $(viewPanel).css('height', ($('body').height() - appBarHeight));
+        $(viewPanel).css('height', ($('body').height() - ($(window).height()*.1)));
```
Comment (Malintha) [inline modules/es-extensions/publisher/extensions/app/greg-diff/themes/default/js/diff.js]: Here we are setting the height of code-mirror view panel height to 90% of the view port height relatively other than hard coding.

Comment (thushara35) [inline modules/es-extensions/publisher/extensions/app/greg-diff/themes/default/js/diff.js]: Please explain if and else part separately. Other than that better if you can create a constant for '768' or else use the comment to explain what it is. Alien would feel hard to maintain the code.

File: modules/es-extensions/store/extensions/app/greg-diff/themes/store/js/diff.js [hunk 2]
```diff
@@ -51,12 +51,12 @@ $('.container-fluid').resize(function(){
...
     if($(window).width() > 768){
-        $(viewPanel).css('height', ($('body').height() - appBarHeight));
+        $(viewPanel).css('height', ($('body').height() - ($(window).height()*.1)));
     }
     else{
-        $(viewPanel).css('height', 'auto');
+        $(viewPanel).css('height', ($('body').height() - ($(window).height()*.1)));
     }
-    $('.CodeMirror, .CodeMirror-merge').height(($(viewPanel).height()-80));
+    $('.CodeMirror, .CodeMirror-merge').height(($(viewPanel).height()-($(window).height()*.1)));
```
Comment (thushara35) [inline modules/es-extensions/store/extensions/app/greg-diff/themes/store/js/diff.js]: Please explain if and else part separately. Other than that better if you can create a constant for '768' or else use the comment to explain what it is. Alien would feel hard to maintain the code.

File: modules/es-extensions/publisher/extensions/app/greg-diff/themes/default/js/diff.js (hunk 1 above)
Comment (Malintha) [inline modules/es-extensions/publisher/extensions/app/greg-diff/themes/default/js/diff.js]: When we allow code mirror to set the height automatically it looks smaller. '768' is referred to the height component of 1024x768 resolution which is considered to be the basic web designing resolution. Therefore, if we detect a screen height larger than 768px, we are obtaining 90% of the screen height responsively for the code mirror divs. If the screen is smaller than 768, we allow code mirror to set it to the default heights.

File: modules/es-extensions/store/extensions/app/greg-diff/themes/store/js/diff.js (hunk 2 above)
Comment (Malintha) [inline modules/es-extensions/store/extensions/app/greg-diff/themes/store/js/diff.js]: When we allow code mirror to set the height automatically it looks smaller. '768' is referred to the height component of 1024x768 resolution which is considered to be the basic web designing resolution. Therefore, if we detect a screen height larger than 768px, we are obtaining 90% of the screen height responsively for the code mirror divs. If the screen is smaller than 768, we allow code mirror to set it to the default heights.

File: modules/es-extensions/publisher/extensions/app/greg-diff/themes/default/js/diff.js (hunk 1 above)
Comment (Malintha) [inline modules/es-extensions/publisher/extensions/app/greg-diff/themes/default/js/diff.js]: Explained in https://github.com/wso2/product-greg/pull/856

**Pull request merged**